app/enterprise/0.35-x/developer-portal/configuration/authentication/oidc.md

```diff
@@ -5,10 +5,10 @@ title: How to Enable OpenId Connect in the Dev Portal
 ### Introduction
 
 The [OpenID Connect Plugin](/hub/kong-inc/openid-connect/) (OIDC)
-allows the Dev Portal to hook into existing authentication setups using third-party 
+allows the Dev Portal to hook into existing authentication setups using third-party
 *Identity Providers* (IdP) such as Google, Yahoo, Microsoft Azure AD, etc.
 
-[OIDC](/hub/kong-inc/openid-connect/) must be used with 
+[OIDC](/hub/kong-inc/openid-connect/) must be used with
```

Comment:
This is false now, yes?

Reply:
I think this is still valid, OIDC's method needs to be session, which is not to be confused with our own session plugin